README.md

```diff
@@ -198,48 +199,20 @@ Options:
   -V, --version
           Print version
 
-RPC options:
-      --http.addr <ADDRESS>
-          Listening address for the http rpc server.
-
-          [default: localhost]
-
-      --http.port <PORT>
-          Listening port for the http rpc server.
-
-          [default: 8545]
-
-      --authrpc.addr <ADDRESS>
-          Listening address for the authenticated rpc server.
-
-          [default: localhost]
-
-      --authrpc.port <PORT>
-          Listening port for the authenticated rpc server.
-
-          [default: 8551]
-
-      --authrpc.jwtsecret <JWTSECRET_PATH>
-          Receives the jwt secret used for authenticated rpc requests.
-
-          [default: jwt.hex]
-
 Node options:
-      --log.level <LOG_LEVEL>
-          Possible values: info, debug, trace, warn, error
-
-          [default: INFO]
-
       --network <GENESIS_FILE_PATH>
-          Alternatively, the name of a known network can be provided instead to use its preset genesis file and include its preset bootnodes. The networks currently supported include holesky, sepolia and ephemery.
+          Alternatively, the name of a known network can be provided instead to use its preset genesis file and include its preset bootnodes. The networks currently supported include holesky, sepolia and mekong.
```

Comment:
This is outdated, we no longer support mekong and do support ephemery

Reply:
yes, fixed here https://github.com/lambdaclass/ethrex/pull/2387